nifi-extension-bundles/nifi-airtable-bundle/nifi-airtable-processors/src/main/java/org/apache/nifi/processors/airtable/QueryAirtableTable.java

```diff
@@ -108,10 +108,11 @@ public class QueryAirtableTable extends AbstractProcessor {
             .required(true)
             .build();
 
-    static final PropertyDescriptor API_KEY = new PropertyDescriptor.Builder()
+    // API Keys are deprecated, Airtable now provides Personal Access Tokens instead.
+    static final PropertyDescriptor PAT = new PropertyDescriptor.Builder()
             .name("api-key")
-            .displayName("API Key")
-            .description("The REST API key to use in queries. Should be generated on Airtable's account page.")
+            .displayName("Personal Access Token")
+            .description("The Personal Access Token (PAT) to use in queries. Should be generated on Airtable's account page.")
```

Comment:
In NiFi 2.0 we can get rid of the old `api-key` property name too. You can use `.name("pat")` and implement `migrateProperties()` method like this:
```
    @Override
    public void migrateProperties(final PropertyConfiguration config) {
        config.renameProperty("api-key", PAT.getName());
    }
```